bee-node/src/inner.rs

```diff
@@ -74,7 +75,7 @@ impl<B: Backend> Node for BeeNode<B> {
             self.worker_stops.remove(&id).unwrap()(&mut self).await;
             for (shutdown, task_fut) in self.tasks.remove(&id).unwrap_or_default() {
                 let _ = shutdown.send(());
-                // TODO: Should we handle thie errpr?
+                // TODO: Should we handle thie error?
```

Comment:
**thie** sorry I'm being super annoying now.

Reply:
I take pride in code. Typos are worth getting rid of!